### ISSUE TITLE
Bulk Actions Not Working > translations_get Function

### ISSUE THREAD

**Issue by baden03** [ReGlot/ReGlot#9]
When submitting multiple translations for Approval using the Bulk Actions feature, the form's action url points to a 404 page not found.  The issue is with the translations_get function https://github.com/fmestrone/ReGlot/blob/master/gp-includes/routes/translation.php#L59
once this function was reverted to the original from GlotPress, the Bulk Actions functioned correctly.

This is a very important feature, as translators can only create translation 'suggestions' and with out bulk approval, an admin (also validator in GlotPress, but it seems this user role been removed in ReGlot?!) must approve each translation one-by-one.

**Comment by fmestrone** [comment]
Hello @baden03,

Sure, bulk actions should work! I haven't spent much time on them as my main objective has been Elgg ;-)

I'll look into it and try and find out what the problem is. What do you mean the validator role has been removed in ReGlot? it should all work as it did before... I have in fact expanded the approval role to work on a locale basis, but this is only available by changing the database for now. i will document it as soon as possible.

Right now I have devoted much time to this, but now i have to go back to doing some commercial work, so i will also to slow down a little... rest assured though that i will keep working on this :-)

**Comment by baden03** [comment]
Yes, I just wanted to list the issues so as not to forget for future fixes.

**Comment by cfoellmann** [comment]
With the patch provided by @baden03 bulk actions are fixed : https://github.com/foe-services/ReGlot/commit/a92008ee38d8fbca9675f4169f8b09c048ffbd39
